src/app/textarea/textarea.directive.unit.spec.ts

```diff
@@ -48,12 +54,44 @@ describe('MzTextareaDirective:unit', () => {
       expect(callOrder[1]).toBe('initElements');
     });
 
+    it('should call initTextareaSubscription method', () => {
+
+      directive.ngOnInit();
+
+      expect(directive.initTextareaSubscription).toHaveBeenCalled();
+      expect(callOrder[2]).toBe('initTextareaSubscription');
+    });
+
     it('should call handleProperties method', () => {
 
       directive.ngOnInit();
 
       expect(directive.handleProperties).toHaveBeenCalled();
-      expect(callOrder[2]).toBe('handleProperties');
+      expect(callOrder[3]).toBe('handleProperties');
+    });
+  });
+
+  describe('ngOnDestroy', () => {
+
+    it('should unsubscribe textareaValueSubscription when subscribed', () => {
+
+      const mockSubscription = new Subscription();
+
+      spyOn(mockSubscription, 'unsubscribe');
+
+      directive.textareaValueSubscription = mockSubscription;
+      directive.ngOnDestroy();
+
+      expect(mockSubscription.unsubscribe).toHaveBeenCalled();
```

Comment:
Change onDestroy unit test to only verify if the subscription is closed.

Reply:
Done